ipatests/test_integration/test_acme.py

```diff
@@ -542,3 +552,129 @@ def install(cls, mh):
 
         tasks.install_client(cls.master, cls.clients[0])
         tasks.install_replica(cls.master, cls.replicas[0])
+
+
+class TestACMERenew(IntegrationTest):
```

Comment:
The TestACME class uses the following:
```
@pytest.mark.skipif(not cainstance.minimum_acme_support(),
                    reason="does not provide ACME")
```
We either need to define the same here or we can remove the skipif from TestACME class (since freeipa.spec file has been updated to required pki 10.10.5).